.github/workflows/ett.yml

```
@@ -117,32 +117,119 @@ jobs:
-
       - name: Container image name
         id: image-name
         uses: ASzc/change-string-case-action@v2
         with:
           # TODO(xlgni): coordinate name change to `env.app-name` with Fenris infrastructure
           string: ghcr.io/${{ github.repository }}-ett-app
-
       - name: Build host container image
         run: docker build ./ --file ./infrastructure/eo/host/Dockerfile --tag ${{ steps.image-name.outputs.lowercase }}:${{ env.host-image-tag }}
-
       - name: Publish host container image
         run: |
           docker login ghcr.io --username ${{ github.actor }} --password ${{ github.token }}
           docker push ${{ steps.image-name.outputs.lowercase }}:${{ env.host-image-tag }}
+
+  container_merge:
+    name: 'Merge: Host container'
+    # if: ${{ github.ref == 'refs/heads/main' }}
+    needs: app
+    outputs:
+      release_tag: ${{ steps.build_and_push.outputs.RELEASE_VERSION }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/download-artifact@v2
+        with:
+          name: ${{ env.app-name }}
+          path: ./dist/apps/ett/${{ env.app-name }}/
+
+      - name: Container image name
+        id: image-name
+        uses: ASzc/change-string-case-action@v2
+        with:
+          # TODO(xlgni): coordinate name change to `env.app-name` with Fenris infrastructure
+          string: ghcr.io/${{ github.repository }}-ett-app
+
+      - name: Build and Push Docker image
+        id: build_and_push
+        uses: Energinet-DataHub/.github/.github/actions/docker-build-and-push@9b79b91f1290f69410eed3ad3855cfe04904f01e
+        with:
+          REGISTRY: ghcr.io
+          DOCKER_USERNAME: ${{ github.actor }}
+          DOCKER_PASSWORD: ${{ github.token }}
+          DOCKER_IMAGE: ${{ steps.image-name.outputs.lowercase }}
+          INCLUDE_LATEST_TAG: true
+          TAG_VERSION_PREFIX: ''
+          TAG_VERSION_SUFFIX: ''
+          MAJOR: 0
+          MINOR: 0
+          PATCH: ${{ github.run_number }}
+          FILE: infrastructure/eo/host/Dockerfile
+
+      - uses: 'marvinpinto/action-automatic-releases@919008cf3f741b179569b7a6fb4d8860689ab7f0'
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          automatic_release_tag: ${{ steps.build_and_push.outputs.RELEASE_VERSION }}
+          prerelease: false
+          draft: false
+          title: Release ${{ steps.build_and_push.outputs.RELEASE_VERSION }}
+
+  update_base_env:
+    name: Update Base Environment
+    runs-on: ubuntu-latest
+    needs:
+      - container_merge
+    env:
+      SERVICE_NAME: eo-frontend
+
+      BASE_ENV_REPO: 'Energinet-DataHub/eo-base-environment'
+      BASE_ENV_REPO_BRANCH: 'development'
+      HELM_CHART_PATH: 'yggdrasil/applications/eo/eo-frontend/eo-frontend.yaml'
+      JOB_URL: '${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}'
+
+    steps:
+      # Checkout the base envionment
```

suggestion: Correct typo
```suggestion
      # Check out the base environment
```